source/declarations.tex

```diff
@@ -2526,15 +2529,19 @@
 \item
 Otherwise, let $S$ be the entity associated with the scope inhabited by
 the \grammarterm{declarator}.
+\end{itemize}
+Then:
+\begin{itemize}
 \item
 If the \grammarterm{declarator} declares
 an explicit instantiation or a partial or explicit specialization,
 the \grammarterm{declarator} does not bind a name.
-If it declares a class member,
-the terminal name of the \grammarterm{declarator-id} is not looked up;
-otherwise, only those lookup results that are nominable in $S$
+If such a \grammarterm{declarator} does not declare a class member,
+the terminal name of the \grammarterm{declarator-id} is looked up,
+and only those lookup results that are nominable in $S$
 are considered when identifying
-any function template specialization being declared\iref{temp.deduct.decl}.
+any function or variable template specialization
```

Comment:
@opensdh Was the different treatment of function and variable template specializations intended?

Reply:
It was (although that doesn't guarantee that it's _good_ to do so!).  The idea is that we do ordinary lookup for the name before the `<` (and for an explicit specialization/instantiation without an explicit template argument list, for consistency): if we find a variable template, it must be the unique lookup result, so there isn't any reason to disqualify that one result because of nominability.  I think we did talk about this in the review, but I can't find the minutes for it.